src/lib/transition_frontier_controller_tests/stubs.ml

```diff
@@ -450,15 +450,45 @@ struct
          (root_breadcrumb |> return |> Quickcheck.Generator.return)
          (gen_breadcrumb ~logger ~trust_system ~accounts_with_secret_keys)
 
-  module Sync_handler = Sync_handler.Make (struct
+  module Best_tip_prover = Best_tip_prover.Make (struct
     include Transition_frontier_inputs
     module Transition_frontier = Transition_frontier
   end)
 
-  module Root_prover = Root_prover.Make (struct
-    include Transition_frontier_inputs
-    module Transition_frontier = Transition_frontier
-  end)
+  module Sync_handler = struct
+    module T = Sync_handler.Make (struct
+      include Transition_frontier_inputs
+      module Transition_frontier = Transition_frontier
+      module Best_tip_prover = Best_tip_prover
+    end)
+
+    let answer_query = T.answer_query
+
+    let get_staged_ledger_aux_and_pending_coinbases_at_hash =
+      T.get_staged_ledger_aux_and_pending_coinbases_at_hash
+
+    let get_transition_chain = T.get_transition_chain
+
+    module Root = T.Root
+
+    (* HACK: This makes it unit tests involving eager bootstrap faster *)
```

Comment:
I don't think call `should_bootstrap` could take a long time. Do we really want this hack? It makes the code look complicated.